sys/arduino/serialport.cpp

```diff
@@ -121,6 +177,48 @@ size_t SerialPort::println(int val, SerialFormat format)
     return (res + 2);
 }
 
+size_t SerialPort::println(unsigned int val)
+{
+    size_t res = print(val);
+    write("\r\n");
```

Comment:
I don't know much about C++, but couldn't this pattern, which is repeated quite often below, be generalized in a private method. It would remove a lot of code duplication.

Reply:
Done